express/io.py

```diff
@@ -31,3 +49,38 @@ def get_file_name(file_uri: str, return_extension=False):
     if return_extension:
         file_name = f"{file_name}{extension}"
     return file_name
+
+
+def create_subprocess_arguments(
+    args: Optional[List[str]] = None, kwargs: Optional[Dict[str, Any]] = None
+) -> List[str]:
```

Comment:
I feel like we're taking quite some assumptions here on how arguments will be parsed, which can often differ between programs. If we want the user to be able to pass in arguments, I would let them pass them in already formatted.